cmd/device-worker/main.go

```diff
@@ -100,32 +100,32 @@ func main() {
 
 	// --- Client metrics configuration ---
 	metricsStore, err := metrics.NewTSDB(dataDir)
-	configManager.RegisterObserver(metricsStore)
+	configManager.RegisterObserver(metricsStore, true)
 	// Metrics Daemon
 	metricsDaemon := metrics.NewMetricsDaemon(metricsStore)
 	workloadMetricWatcher := metrics.NewWorkloadMetrics(metricsDaemon)
-	configManager.RegisterObserver(workloadMetricWatcher)
+	configManager.RegisterObserver(workloadMetricWatcher, true)
 
 	wl, err := workload2.NewWorkloadManager(dataDir, deviceId)
 	if err != nil {
 		log.Fatalf("cannot start Workload Manager. DeviceID: %s; err: %v", deviceId, err)
 	}
-	configManager.RegisterObserver(wl)
+	configManager.RegisterObserver(wl, true)
 	wl.RegisterObserver(workloadMetricWatcher)
 
 	hw := hardware2.Hardware{}
 
 	dataMonitor := datatransfer.NewMonitor(wl, configManager)
 	wl.RegisterObserver(dataMonitor)
-	configManager.RegisterObserver(dataMonitor)
+	configManager.RegisterObserver(dataMonitor, false)
```

Comment:
@jakub-dzon  about this one, I'm not sure if we should wait until get the new s3 config, it's not clear to me.

Reply:
Right now it should be `false`, but we should adjust `Monitor.Start()` to not use `ForceUpdate` and set it to `true` then